bin/mini-langstream/README.md

```diff
@@ -0,0 +1,40 @@
+# mini-langstream
```

Comment:
I am not sure that the README should go into the "bin" directory,
maybe it is better to move the whole directory to the root (not as child of "bin")

Reply:
good point